src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/adsbase/AdsBaseObject.java

```diff
@@ -496,6 +498,38 @@ public boolean checkSlotOnPageLoaded(String slotName) {
       restoreDefaultImplicitWait();
     }
   }
+  /**
+   * Check if AdEngine loaded the ad web elements inside slot with mobile state
+   */
+  public boolean checkSlotOnPageLoaded(String slotName, Boolean isMobile) {
```

Comment:
Consider making base method (without Boolean param) to call this one. This way you would eliminate code duplication.